ng-ncc-app/src/app/components/agent/agent.component.scss

```diff
@@ -0,0 +1,33 @@
+$agent-avatar-size: 3rem !default;
+$agent-spacing: 2rem !default;
+$agent-avatar-spacing: .5rem !default;
+$agent-text-width: 15rem !default;
+
+////////////////////
+
+:host {
+    color: inherit;
+    display: inline-flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    margin-left: $agent-spacing;
+}
+
+.agent__avatar {
+    width: $agent-avatar-size;
+    height: $agent-avatar-size;
+    margin-right: $agent-avatar-spacing;
+}
+
+.agent__text {
+    color: inherit;
+    margin: 0;
+    min-width: $agent-text-width;
+}
+
+.agent__name {
+    color: inherit;
+    display: block;
+    font-weight: bold;
```

Comment:
can we use 700 instead of bold here and in other places?

Reply:
Why 700 instead of bold?

Reply:
Back in the days some browsers used to interpret bold in different ways. Now is not the case anymore. In general I prefer numbers compared to words because gives more granular control considering you can have font weight from 100 to 900 if the font in use comes with all those variants and visually I find it easier to understand that 700 (bold) means bigger than 400 (normal). Anyway, that's just a personal preference so feel free to leave bold. 😄